packages/edit-site/src/components/global-styles/font-families.js

```diff
@@ -52,7 +52,7 @@ function FontFamilies() {
 			<VStack spacing={ 4 }>
 				{ themeFonts.length > 0 && (
 					<VStack>
-						<Subtitle level={ 3 }>{ __( 'Theme Fonts' ) }</Subtitle>
+						<Subtitle level={ 3 }>{ __( 'Theme' ) }</Subtitle>
```

Comment:
This seems like a place where we should use `_x` and/or a translator comment to provide a context for translation. Translating just the word "Theme" or "Custom" in isolation may be challenging.

```suggestion
						/* translators: Heading for a list of fonts provided by the theme. */
						<Subtitle level={ 3 }>{ _x( 'Theme', 'font source' ) }</Subtitle>
```